website/src/routes/api/(schemas)/string/index.mdx

```diff
@@ -2,6 +2,116 @@
 title: string
 ---
 
+import { Property } from '~/components';
+import { Link } from '@builder.io/qwik-city';
+
 # string
 
-> The content of this page is not yet ready. Until then just use the [source code](https://github.com/fabian-hiller/valibot/blob/main/library/src/schemas/string/string.ts).
+Schema that parses the input to string.
+
+```ts
+const schema = string(errorMessage, pipe);
+```
+
+## Parameter
+
+- `errorMessage` <Property type='string'/> = 'Invalid type'
+- `pipe` <Property type={{ type: 'custom', name: 'Pipe<string>' }}/> = []
+
+## Return
+
+- `schema` <Property type={{ type: 'custom', name: 'StringSchema' }}/>
+
+## Examples
```

Comment:
Here I would make only 2 to 3 concrete examples. We can link the other possibilities below.

```ts
// Schema to validate an eamil
const emailSchema = string([
  minLength(1, 'Please enter your email.'),
  email('The email is badly formatted.'),
  maxLength(30, 'Your email is to long.'),
]);

// Example 2 ...
```